assets/js/base/components/price-slider/stories/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { React, useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PriceSlider from '../';
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/PriceSlider',
+	component: PriceSlider,
+};
+
+export const Default = () => {
+	// PriceSlider expects client to update min & max price
+	// is this a bug? not fully self-contained
+	const [ min, setMin ] = useState( 1000 );
+	const [ max, setMax ] = useState( 5000 );
+	return (
+		<div>
```

Comment:
Why the `div` container? It shouldn't be needed here.

Reply:
Woops, goneburgers 🍔 fec4ba4af6565668e4439268b34e8eceee60ca7c